libass/ass_shaper.c

```diff
@@ -411,11 +438,21 @@ static hb_font_t *get_hb_font(ASS_Shaper *shaper, GlyphInfo *info)
     hb_fonts = font->shaper_priv->fonts;
     if (!hb_fonts[info->face_index]) {
         FT_Face face = font->faces[info->face_index];
-        hb_font_t *hb_font = hb_fonts[info->face_index] =
-            hb_ft_font_create(face, NULL);
+        hb_face_t *hb_face = hb_face_create_for_tables(get_reference_table, face, NULL);
```

Comment:
Note that `hb_face_create_for_tables()` never return NULL. In case of failure it will return a singleton empty `hb_face_t` (as returned by `hb_face_get_empty()`).